servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/LoadBalancingPolicy.java

```diff
@@ -36,5 +36,6 @@ interface LoadBalancingPolicy<ResolvedAddress, C extends LoadBalancedConnection>
      * @param targetResource the name of the target resource, useful for debugging purposes.
      * @return a {@link HostSelector}
      */
-    HostSelector<ResolvedAddress, C> buildSelector(List<Host<ResolvedAddress, C>> hosts, String targetResource);
+    <T extends C> HostSelector<ResolvedAddress, T> buildSelector(
```

Comment:
This is an unfortunate artifact of the `LoadBalancerFactory` having similarly generic methods on it. Those methods are deprecated but are still the main entry point to the load balancer.